TF_Service_dotNet/TouchFree.Tooling/ToggleablePlugin.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Ultraleap.TouchFree.Tooling
+{
+    // Struct: ToggleablePlugin
+    // A Data structure used to toggle the use of plugins.
+    [System.Serializable]
+    internal struct ToggleablePlugin
```

Comment:
As @MattGrayUL said elsewhere, these toggleable plugins were just a way of providing an easy UI in the Unity version of the tooling. I don't see any value in them in a C# tooling library